src/pages/Lobby/components/PlayersTable/PlayersTable.tsx

```diff
@@ -0,0 +1,24 @@
+import { Position, Row } from './styles';
+import { PlayersTableProps } from './types';
+
+export function PlayersTable({ players }: PlayersTableProps) {
+  function getPlayerNameItem(current: boolean, name: string) {
+    if (!current) {
+      return name;
+    }
+    return 'Você';
+  }
+
+  return (
+    <table>
+      <tbody>
```

Comment:
**Question:** Why did you choose to use the table?

Reply:
I wanted the first item (the position) to have a fixed width to be well positioned on the screen independently of the value. Do you see a better approach?

Reply:
very interesting, seems fair